src/gui/Player.ts

```diff
@@ -145,9 +145,8 @@ class Player extends EventEmitter {
 
         this.useExternalTransport = () => {
             // eslint-disable-next-line no-restricted-globals
-            const useExternal =
-                new URLSearchParams(parent.location.search).getAll("noUi")
-                    .length > 0 || this._controller.options?.noUi
+            const useExternal = 
+                getSetting("noUi") === "true" || this._controller.options?.noUi
```

Comment:
In the storyplayer harness plugin (https://github.com/bbc/rd-ux-storyplayer-harness/blob/0fba3592def9540b835efe2ef0a2535af9f40b0f/plugin/plugin.js#L31C1-L31C84), to get the URL of the enclosing page we use:
`const pageUrl = new URL(pluginUtils.playerInterface.settings.embedPageURL);`

Not sure if that makes any difference to how we do things

Reply:
Yes, but this is meaningless outside the plugin.

Reply:
Ah yeah of course